src/components/item/form/AppForm.tsx

```diff
@@ -83,68 +77,68 @@ const AppForm = ({
           } as Partial<DiscriminatedItem>
         }
       />
+      <br />
 
-      {isAppsLoading ? (
-        <Skeleton height={60} />
-      ) : (
-        <Autocomplete
-          id={ITEM_FORM_APP_URL_ID}
-          options={data?.toArray() ?? []}
-          getOptionLabel={(option) => {
-            if (typeof option === 'string') {
-              return option;
-            }
-            return option.url;
-          }}
-          filterOptions={(options, state) => {
-            const filteredOptionsByName = options.filter((opt: AppRecord) =>
-              opt.name.toLowerCase().includes(state.inputValue.toLowerCase()),
-            );
-            return filteredOptionsByName;
-          }}
-          value={value}
-          clearOnBlur={false}
-          onChange={handleAppSelection}
-          onInputChange={handleAppInput}
-          renderOption={(
-            props: HTMLAttributes<HTMLLIElement>,
-            option: AppRecord,
-          ) => (
-            <li
-              // eslint-disable-next-line react/jsx-props-no-spreading
-              {...props}
-              style={{
-                display: 'flex',
-                padding: 8,
-                alignItems: 'center',
-              }}
-              id={buildItemFormAppOptionId(option.name)}
-            >
-              <img
-                style={{
-                  verticalAlign: 'middle',
-                  margin: 8,
-                  height: '30px',
-                }}
-                src={option.extra?.image as string}
-                alt={option.name}
-              />
-              <Typography variant="body1" pr={1}>
-                {option.name}
-              </Typography>
-              <Typography variant="caption">{option.description}</Typography>
-            </li>
-          )}
-          renderInput={(params) => (
+      {isAppsLoading && <Skeleton height={60} />}
+
+      {!isAppsLoading &&
+        (isNewApp ? (
+          <Box sx={{ marginTop: '24px' }}>
```

Comment:
I think the following works:
```suggestion
          <Box sx={{ mt: 3 }}>
```